store/boltdbStore/service.go

```diff
@@ -173,7 +173,7 @@ func (ss *serviceStore) GetSourceServiceToken(name string, namespace string) (*m
 	var out model.Service
 	s, err := ss.getServiceByNameAndNs(name, namespace)
 	switch {
-	case err == sql.ErrNoRows:
```

Comment:
here maybe add new if code block after `err != nil` will be better, not modify original code.
```
case s == nil:
        return nil, nil
```